pom.xml

```diff
@@ -66,9 +66,11 @@ under the License.
   <properties>
     <javaVersion>7</javaVersion>
     <mdoVersion>2.1.0</mdoVersion>
-    <mavenArchiverVersion>3.5.1</mavenArchiverVersion>
-    <mavenFilteringVersion>3.2.0</mavenFilteringVersion>
-    <mavenVersion>3.0</mavenVersion>
+    <mavenVersion>3.2.5</mavenVersion>
+    <sisuVersion>0.3.5</sisuVersion>
+    <slf4jVersion>1.7.5</slf4jVersion>
```

Comment:
1.7.5?

Reply:
maven 3.2.5 used that version, yes, just like aether and not "maven resovler"